mevolib/align/_Mafft.py

```diff
@@ -20,7 +20,7 @@
 SPRT_INFILE_FORMATS = ['fasta']
 INFILE_CMD = ''
 KEYWORDS = {
-    'default': ['--auto', '--quiet', '--thread', str(NUMCORES)],
+    'default': ['--anysymbol', '--auto', '--quiet', '--thread', str(NUMCORES)],
```

Comment:
Is there a reason for this? Based on the documentation, this parameter is a rather unusual requirement and I do not think it should be part of the `default` configuration.